platformio.ini

```diff
@@ -805,3 +806,13 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+# ------------------------------------------------------------------------------
+# Arduino® Nano ESP32
+# ------------------------------------------------------------------------------
+[env:arduino_nano_esp32]
+platform = https://github.com/platformio/platform-espressif32.git
```

Comment:
should be `platform = espressif32@6.4.0`

Reply:
I tried that, but it causes the build to fail, looking for something called `FrameworkArduinoVariant`.

Reply:
Apparently, requesting two builds back to back resolves the `FrameworkArduinoVariant` error. Odd.